EIPS/eip-4747.md

```diff
@@ -0,0 +1,313 @@
+
+The complete removal of all empty accounts ensures that certain edgecases of EIP-161 can never occur on Ethereum Mainnet. Continuing to define and test those cases as part of the Ethereum Specification burdens future client implementors with unnecessary technical debt. This EIP declares those cases undefined and leaves clients free to assume they will not occur.
+
+## Backwards Compatibility
+
+This EIP is identical to EIP-161 except for the following differences, none of which affect Mainnet. The differences are:
+
+### "Potentially state-changing operations"
+
+EIP-161 specifies 11 "potentially state-changing operations" that trigger state clearing. All but the 3 listed in this EIP are irrelevant, for the following reasons:
+
+#### Impossible
+
+* Receiving zero value mining reward/fees (this would become possible after the merge).
+
+#### Cannot happen to an empty account
+
+* Being the source of a `CREATE`.
+* Being the source of a `CALL`.
+* Being refunded by a `SELFDESTRUCT`
+
+#### Causes the account to become non-empty
+
+* Being the sender of a message call transaction.
+* Being the sender of a contract creation transaction.
+* Being created by a `CREATE`.
+* Being created by a contract creation transaction.
+
+### Interaction with `STATICCALL`
+
+The interaction between `STATICCALL` and account clearing has never been specified in an EIP. The Ethereum currently testsuite requires that `STATICCALL` triggers state clearing.
+
+This EIP formally undefines all interactions between `STATICCALL` and state clearing as it has never happened on Mainnet and cannot happen in future.
+
+### "At the end of the transaction"
+
+This only makes a difference if an account is deleted and later recreated in the same transaction. This never happens on Mainnet.
+
+### Test Cases
+
+All test cases involving empty accounts in the Ethereum execution layer test suite shall be removed unless they relate to the Spurious Dragon Hardfork. If a Spurious Dragon test relates involved deprecated edgecase the test must be removed or reworked.
+
+### Other networks
+
+Ropsten had empty accounts seeded at genesis. They appear to have been cleared early in Ropsten's history before the Byzantium Hardfork. Ropsten has never been checked for edgecases occurring. All other Ethereum testnets have had EIP-161 from genesis.
+
+As a security precaution all empty accounts on Ethereum Classic have been cleared, but no checks for edgecases occurring have been done. Due to EIP-161's age the vast majority of EVM compatible networks have supported it from genesis.
+
+## Security considerations
+
+This EIP is only equivalent to EIP-161 on Mainnet if the following facts and must not be finalised until they have been independently verified:
```

Comment:
```suggestion
This EIP is only equivalent to EIP-161 on Ethereum Mainnet if the following facts are true:
```
EIPs should be written in their final form, which is presumably *after* independent verification has occurred.  I recommend mentioning the need for independent verification in the discussions-to link.